Source/JavaScriptCore/wasm/js/WebAssemblyFunction.cpp

```diff
@@ -351,22 +363,29 @@ CodePtr<JSEntryPtrTag> WebAssemblyFunction::jsCallEntrypointSlow()
 #if !CPU(ARM) // ARM has no pinned registers for Wasm Memory, so no need to set them up
     if (!!instance()->instance().module().moduleInformation().memory) {
         GPRReg baseMemory = pinnedRegs.baseMemoryPointer;
-        GPRReg scratchOrBoundsCheckingSize = InvalidGPRReg;
         auto mode = instance()->memoryMode();
 
-        if (isARM64E()) {
+        // Size and basePointer will not be changed.
```

Comment:
Let's say `Capacity` instead of `Size`.  We're loading `mappedCapacity` below.  I understand that it's being used for the bounds check size, but ultimately, it's because the capacity doesn't change.

Also, should we move this comment to line 370".  It only applies to the `if` case, right?

Reply:
Changed.